rsts/deployment/plugin_setup/k8s/index.rst

```diff
@@ -1,57 +1,414 @@
 .. _deployment-plugin-setup-k8s:
 
-################
-K8s Plugin Setup
-################
 
+K8s Operator backend Plugin Setup
```

Comment:
```suggestion
K8s Operator
```
Since the main heading is "Plugin setup", I think K8s Operator should work.

Reply:
Can you also modify AWS plugins setup to AWS, etc.?
<img width="318" alt="image" src="https://user-images.githubusercontent.com/27777173/145156131-afeb6ea1-9ef8-4969-864a-cf5d94f2aa8c.png">